db/seeds.rb

```diff
@@ -250,6 +250,7 @@
   welcome_thread: "Sloan here again! 👋 DEV is a friendly community. Why not introduce yourself by leaving a comment in <a href='/welcome'>the welcome thread</a>!",
   twitter_connect: "You're on a roll! 🎉 Let's connect your <a href='/settings'> Twitter account</a> to complete your identity so that we don't think you're a robot. 🤖",
   github_connect: "You're on a roll! 🎉 Let's connect your <a href='/settings'> GitHub account</a> to complete your identity so that we don't think you're a robot. 🤖",
+  customize_feed: "Hi, it's me again! 👋Now that you're a part of the DEV community, let's focus on personalizing your content. You can start by <a href='/tags'> following some tags </a> to help customize your feed! 🎉",
```

Comment:
Looks like our convention is a space after the emoji, we should probably do that here as well.

Reply:
Nice catch!
```suggestion
  customize_feed: "Hi, it's me again! 👋 Now that you're a part of the DEV community, let's focus on personalizing your content. You can start by <a href='/tags'> following some tags </a> to help customize your feed! 🎉",
```